integration-tests/suites/symlink_process.go

```diff
@@ -75,5 +68,5 @@ func (s *SymbolicLinkProcessTestSuite) TestSymbolicLinkProcess() {
 	assert.Equal(s.T(), endpoints[0].Originator.ProcessName, lnProcess.Name)
 	assert.Equal(s.T(), endpoints[0].Originator.ProcessExecFilePath, lnProcess.ExePath)
 	assert.Equal(s.T(), endpoints[0].Originator.ProcessArgs, lnProcess.Args)
-	assert.Equal(s.T(), 9092, endpoints[0].Address.Port)
+	assert.Equal(s.T(), 5000, endpoints[0].Address.Port)
```

Comment:
[nit] could do with a small comment describing why it's this port number, because it's not clear in the test that it's the port the image exposes

Reply:
Makes sense, I'll add it.